app/lib/pre_assembly/bundle.rb

```diff
@@ -39,11 +39,14 @@ class BundleContext
 
     # Unpack the user-supplied parameters, after converting
     # all hash keys and some hash values to symbols.
-    def initialize(params = {})
+    def initialize(params)
       params = Assembly::Utils.symbolize_keys params
-      Assembly::Utils.values_to_symbols! params[:project_style]
-      cmc          = params[:content_md_creation]
-      cmc[:style]  = cmc[:style].to_sym
+      raise ArgumentError, ':bundle_dir is required' unless params[:bundle_dir] # TODO: replace w/ AR validation
+      [:content_md_creation, :object_discovery, :project_style, :stageable_discovery].each { |k| params[k] ||= {} }
+      params[:project_style].transform_values! { |v| v.is_a?(String) ? v.to_sym : v }
+      params[:project_style][:content_structure] ||= :simple_image
+      params[:content_md_creation][:style] ||= :default
+      params[:content_md_creation][:style] &&= params[:content_md_creation][:style].to_sym
```

Comment:
Since you've just set `params[:content_md_creation][:style] ||= :default` wouldn't this always be truthy and thus the `&&=` couldn't ever be false?  If that was the case couldn't you just set to simple assignment (e.g. `=`)

Reply:
Yes, that's correct.